shell/common/rasterizer_unittests.cc

```diff
@@ -76,7 +77,7 @@ class MockExternalViewEmbedder : public ExternalViewEmbedder {
 }  // namespace
 
 TEST(RasterizerTest, create) {
-  MockDelegate delegate;
+  NiceMock<MockDelegate> delegate;
```

Comment:
These changes are not essential to this patch, but eliminate a lot of noise when running these test targets related to unexpected mock function calls.